app/src/behaviors/behavior_backlight.c

```diff
@@ -20,21 +20,59 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 static int behavior_backlight_init(const struct device *dev) { return 0; }
 
+static int
+on_keymap_binding_convert_central_state_dependent_params(struct zmk_behavior_binding *binding,
+                                                         struct zmk_behavior_binding_event event) {
```

Comment:
I see no reason to add this function. You can do all the checks inside `on_keymap_binding_pressed.` Adding this function only increases the complexity and readability of the code.

Reply:
This is for the split locality support. When a key is pressed on the peripheral it synchronises the two sides. As is the code can end up in a state where one side is on and the other is off and pressing the toggle key will turn one side on and the other off, and the brightness will be out of sync

Reply:
I understand, my bad